PendingReleaseNotes

```diff
@@ -47,120 +50,128 @@
+  message bus (for example, RabbitMQ, Kafka) for that purpose.
+
+* RGW: The serialized format of notification and topics has been changed, 
+  so that new and updated topics will be unreadable by old RGWs. We recommend
+  completing the RGW upgrades before creating or modifying any notification
+  topics.
+* RBD: Trailing newline in passphrase files (for example: the 
+  ``<passphrase-file>`` argument of the ``rbd encryption format`` command and
+  the ``--encryption-passphrase-file`` option of other commands) is no longer
+  stripped.
+* RBD: Support for layered client-side encryption has been added. It is now 
+  possible to encrypt cloned images with a distinct encryption format and
+  passphrase, differing from that of the parent image and from that of every
+  other cloned image. The efficient copy-on-write semantics intrinsic to
+  unformatted (regular) cloned images have been retained.
+* CEPHFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* The ``perf dump`` and ``perf schema`` commands have been deprecated in 
+  favor of the new ``counter dump`` and ``counter schema`` commands. These new
+  commands add support for labeled perf counters and also emit existing
+  unlabeled perf counters. Some unlabeled perf counters became labeled in this
+  release, and more will be labeled in future releases; such converted perf
+  counters are no longer emitted by the ``perf dump`` and ``perf schema``
+  commands.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+* The ``active_clients`` array displayed by the ``ceph mgr dump`` command now
+  has a ``name`` field that shows the name of the mgr module that registered a
+  RADOS client. Previously, the ``active_clients`` array showed the address of
+  a module's RADOS client, but not the name of the module.
+* RBD: All rbd-mirror daemon perf counters have become labeled and are now
+  emitted only by the new ``counter dump`` and ``counter schema`` commands. As
+  part of the conversion, many were also renamed in order to better
+  disambiguate journal-based and snapshot-based mirroring.
+* RBD: The list-watchers C++ API (``Image::list_watchers``) now clears the 
+  passed ``std::list`` before appending to it. This aligns with the semantics
+  of the C API (``rbd_watchers_list``).
+* Telemetry: Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard.  For more details, see:
```

Comment:
@zdover23 - A double space seems to have been introduced during the line wrap.

Reply:
Accepted.